helion/_compiler/device_ir.py

```diff
@@ -129,9 +129,9 @@ def _get_proxy_slot(
         ),
     ):
         current_location().set_fx_location()
-        return proxy_tensor.make_fx(fn, decomposition_table=select_decomp_table())(
-            *args
-        ).graph
+        decomp_table = select_decomp_table()
+        decomp_table.pop(torch.ops.aten.stack.default, None)
+        return proxy_tensor.make_fx(fn, decomposition_table=decomp_table)(*args).graph
```

Comment:
Normally, `torch.stack` is decomposed to unsqueeze + cat, but I haven’t figure out a way to make `codegen_cat` work, so as a workaround we disable the decomp for torch.stack and implement `codegen_stack` instead.